examples/visualization/plot_evoked_arrowmap.py

```diff
@@ -0,0 +1,30 @@
+"""
+========================================
+Plotting topographic arrowmaps of evoked data
+========================================
+
+Load evoked data and plot topomaps for selected time points.
```

Comment:
please mention the arrows and explain what they will represent

Reply:
Done